lib/core/connector_settings.rb

```diff
@@ -17,7 +17,7 @@ class ConnectorSettings
     DEFAULT_REQUEST_PIPELINE = 'ent-search-generic-ingestion'
     DEFAULT_EXTRACT_BINARY_CONTENT = true
     DEFAULT_REDUCE_WHITESPACE = true
-    DEFAULT_RUN_ML_INFERENCE = false
+    DEFAULT_RUN_ML_INFERENCE = true
```

Comment:
Should this change be here?

Reply:
It went away after rebasing. I guess it was a side effect.